client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -1,84 +1,44 @@
 /**
- * External dependencies
+ * External Dependencies
```

Comment:
Are you sure it should be upper case? The lowercase version seems the most common way of writing it in Calypso.